TwitchLib.PubSub/TwitchPubSub.cs

```diff
@@ -803,6 +821,13 @@ public void ListenToChatModeratorActions(string myTwitchId, string channelTwitch
             ListenToTopic(topic);
         }
 
+        public void ListenToUserModerationNotifications(string myTwitchId, string channelTwitchId)
```

Comment:
maybe rather call it userId instead of myTwitchId and just channelId than channelTwitchId it always feels repetetive to include twitch in variable names

Reply:
This was done to be consistent with existing code. But ya, I agree. Will update.